src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -205,6 +205,8 @@ internal override void ResetConsoleLoggerState()
             _hasBuildStarted = false;
 
             // Reset the two data structures created when the logger was created
+            target_framework_mapping = new Dictionary<(int, int), string>();
+            target_framwork_errorwarning = new Dictionary<string, (int warningCount, int errorCount)>();
```

Comment:
nit:
```suggestion
            target_framework_errorwarning = new Dictionary<string, (int warningCount, int errorCount)>();
```
And that will have to be copied below.